ssm/ssm.go

```diff
@@ -8,13 +8,10 @@ import (
 	"github.com/aws/aws-sdk-go/service/ssm/ssmiface"
 )
 
-func ProcessParameters(svc ssmiface.SSMAPI, parameters map[string]string, environment string, verbose bool) {
-
-	fmt.Printf("Processing %d parameters for %s environment\n", len(parameters), environment)
-
+func ProcessParameters(svc ssmiface.SSMAPI, parameters map[string]string, verbose bool) {
 	for k, v := range parameters {
 		if verbose {
-			fmt.Printf("Putting and tagging parameter with key \"%s\" and value \"%s\"\n", k, v)
+			fmt.Printf("**PUTTED** \"%s\" - \"%s\"\n", k, v)
```

Comment:
Didn't use "CREATED" because the entry can be updated, if it already exists. I chose "Putted" because the equivalent CLI command is "aws put-parameter".